schemas/test/en/test_supplementary_data_non_list_items.json

```diff
@@ -0,0 +1,503 @@
+                                "answers": [
+                                    {
+                                        "id": "new-email-answer",
+                                        "label": "Contact email",
+                                        "mandatory": true,
+                                        "type": "TextField"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "sales-breakdown-block",
+                            "question": {
+                                "id": "sales-breakdown-question",
+                                "title": {
+                                    "text": "How much of the {sales} total UK sales was from Bristol and London?",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "sales",
+                                            "transforms": [
+                                                {
+                                                    "transform": "format_currency",
+                                                    "arguments": {
+                                                        "number": {
+                                                            "source": "supplementary_data",
+                                                            "identifier": "total_uk_sales"
+                                                        }
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                },
+                                "type": "Calculated",
+                                "warning": "These answers must not exceed the reported total from the supplementary data",
+                                "calculations": [
+                                    {
+                                        "calculation_type": "sum",
+                                        "value": {
+                                            "source": "supplementary_data",
+                                            "identifier": "total_uk_sales"
+                                        },
+                                        "answers_to_calculate": ["sales-bristol-answer", "sales-london-answer"],
+                                        "conditions": ["less than", "equals"]
+                                    }
+                                ],
+                                "answers": [
+                                    {
+                                        "id": "sales-bristol-answer",
+                                        "label": "Bristol Sales",
```

Comment:
We only recently added placeholder in label, worth adding supplementary data here.